src/shared/ThisContext.js

```diff
@@ -202,22 +235,29 @@ export default class UIHContext {
   /**
    * Loads the given component.
    *
-   * @param component:  The component Type
-   *                    or created component element (eg: <MyComponent/>).
-   *
-   * @param props:      Optional. The component props
-   *                    (if not passed in with a component element).
-   *
-   * @param children:   Optional. The component children
-   *                    (if not passed in with a component element).
-   *
+   * @param component:  The component Type (e.g. MyComponent)
+   *                    or created component element (e.g.: <MyComponent/>).
    */
-  load(component, props, children) {
-    if (!component) {
-      if (isBrowser) { log.warn('Cannot load: a component was not specified (undefined/null)'); }
-    } else {
-      api.loadComponent(component, props, children);
-    }
+  component(component) {
+    invariant(component, 'Cannot load: a component was not specified (undefined/null)');
```

Comment:
I'm liking `invariant` already :)
Much cleaner to ready than IF statements as parameter guards.